_pydevd_bundle/pydevd_cython.pyx

```diff
@@ -1396,7 +1396,7 @@ def should_stop_on_exception(py_db, PyDBAdditionalThreadInfo info, frame, thread
             exception_breakpoint = None
             try:
                 if py_db.plugin is not None:
-                    result = py_db.plugin.exception_break(py_db, frame, thread, arg)
+                    result = py_db.plugin.exception_break(py_db, frame, thread, arg, is_unwind)
```

Comment:
This is a different way to solve the django issue. It checks to make sure we're not during an unwind. In 3.11 this isn't a problem because the cached exception prevents the unwind event from firing.